test/integration/trail_page_test.rb

```diff
@@ -192,7 +192,7 @@ def setup
 
       assert page.has_link? "reviewde Leffe Blond", href: review_path(r)
       assert page.has_link? "maakte een review van Leffe Blond", href: review_path(r)
-      assert page.has_link? "wijzigde een review van Leffe Blond", href: review_path(r), count: 2
+      assert page.has_link? "wijzigde een review van Leffe Blond", href: review_path(r), count: 3
```

Comment:
Where is this change coming from?

Reply:
When running `bundle update`, a couple of errors occurred during testing. 
Weirdly enough this is one of them. I have no idea why it is different

Reply:
We have tests to catch weird stuff, also during upgrading gems so this should be figured out instead of just changed in the test.